.github/workflows/lint.yml

```diff
@@ -3,7 +3,19 @@ name: lint
 on:
   push:
     branches: [ master ]
+    paths:
```

Comment:
lint is fast enough that this path restriction is not necessary. It's too wordy and the benefit is just not enough.